ts/src/coinbase.ts

```diff
@@ -1522,6 +1532,25 @@ export default class coinbase extends Exchange {
         return result;
     }
 
+    market (symbol: string): MarketInterface {
+        // as they are aliases, we need to return the original market
+        const originalInput = symbol;
```

Comment:
again, I don't love this approach, because if the user loads markets from cache this won't work.

Reply:
ok, i'll rewrite